src/index.ts

```diff
@@ -92,6 +110,24 @@ export class PetraWallet implements AdapterPlugin {
     }
   }
 
+  async signTransaction(
+    transaction: Types.TransactionPayload | TxnBuilderTypes.TransactionPayload,
+  ): Promise<{ hash: Types.HexEncodedBytes }> {
+    try {
+      // TODO: We should update the wallet adapter to support options
+      const response = await (window.petra as any).signTransaction(
```

Comment:
nit: `await (this.provider as any).signTransaction`

Reply:
typing issue :)

Reply:
it's not on provider